src/trace/trace.c

```diff
@@ -314,6 +317,22 @@ void trace_log_filtered(bool send_atomic, const void *log_entry, const struct tr
 	va_end(vl);
 }
 
+#elif CONFIG_SUECREEK
+
+void trace_log_filtered(bool send_atomic, const void *log_entry,
+			const struct tr_ctx *ctx, uint32_t lvl,
+			uint32_t id_1, uint32_t id_2, int arg_count, ...)
+{
+}
+
+void trace_log_unfiltered(bool send_atomic, const void *log_entry,
+			  const struct tr_ctx *ctx, uint32_t lvl,
+			  uint32_t id_1, uint32_t id_2, int arg_count, ...)
+{
+}
+
+#endif
```

Comment:
```suggestion
#endif /* CONFIG_TRACE_DMA */
```
This is a huge #ifdef

Reply:
yes, thanks, updated